src/platform/P6/args.gni

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("//build_overrides/p6.gni")
+
+import("${chip_root}/examples/platform/p6/args.gni")
+
+# ARM architecture flags will be set based on p6_family.
+arm_platform_config = "${p6_sdk_build_root}/p6_arm.gni"
+
+mbedtls_target = "${p6_sdk_build_root}:p6_build"
+
+chip_device_platform = "p6"
+treat_warnings_as_errors = false
```

Comment:
Please don't enable this for your entire platform. The build should be warnings clean, so that we don't introduce new warnings.

Reply:
please note, removed this for build and added compiler options only for p6_sdk builds